.github/assets/ISSUE_TEMPLATE/new-tweet-request.md

```diff
@@ -0,0 +1,14 @@
+---
+name: New tweet request
+about: Create a new tweet request
+title: "[Please change the title]"
+labels: ''
+assignees: ''
+
+---
+
+**Provide the actual tweet content below**
+<!-- Input your tweet content exactly below the commented colons -->
```

Comment:
I believe "between the commented colons" would help. I hope people don't start putting tweets below the second commented colon. :)

Reply:
good catch! modified the statement now as suggested.